mayavi/__init__.py

```diff
@@ -5,7 +5,7 @@
     Part of the Mayavi project of the Enthought Tool Suite.
 """
 
-__version__ = '4.3.2'
+__version__ = '5.0.0'
```

Comment:
Hey Deepak,

Thanks for doing all this! It's really great to know that Mayavi
development is alive and kicking! This software is really widely used.